### ISSUE TITLE
Plus doesn't disappear when creating a code cell

### ISSUE THREAD

**Issue by adamwiggins** [inkandswitch/livebook#28]
Keyboard or clicking the plus produces same result.

<img width="219" alt="screen shot 2016-01-20 at 16 49 25" src="https://cloud.githubusercontent.com/assets/177/12453960/c901c632-bf95-11e5-901f-24174ff703a3.png">

**Comment by brettimus** [comment]
To clarify: The plus _should_ appear when you create a code cell?

**Comment by adamwiggins** [comment]
Oops, let me rephrase more clearly:

* Desired behavior: the plus is only visible when your cursor is at the start of a blank prose line.
* Current behavior: creating a cell does not cause the plus to disappear.

**Comment by brettimus** [comment]
:+1:

**Comment by luciasantamaria** [comment]
Can't reproduce in Chrome in Mac. Happens in Firefox in Mac though.

**Comment by adamwiggins** [comment]
Confirmed that Chrome has the desired behavior, Firefox exhibits the problem.
